src/components/pages/user-registration-page/user-registration-page.tsx

```diff
@@ -26,6 +26,7 @@ function validatePhone(phone: string) {
 }
 
 type UserRegisterDataWithValidation = {
+  /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
```

Comment:
It looks like this isn't necessary. I'm not getting any errors when removing it.

Reply:
I agree... but for whatever reason - I was getting an error. So weird.